susemanager-utils/susemanager-sls/src/modules/mgractionchains.py

```diff
@@ -128,8 +145,15 @@ def start(actionchain_id):
     except Exception as exc:
         log.error("There was an error while syncing custom states and execution modules")
 
-    inside_transaction = os.environ.get("TRANSACTIONAL_UPDATE")
-    if __grains__.get("transactional") and not inside_transaction:
+    reboot_required = False
+    inside_transaction = False
+    if __grains__.get("transactional"):
```

Comment:
`__grains__.get("transactional")` is repetative, I would prefer to put it to the local variable firtst and check for it after.